src/app/core/user/user.service.ts

```diff
@@ -280,6 +280,7 @@ export class UserService {
           if (
             typeof updateParameters.checkTrigger === `number` &&
             updateParameters.checkTrigger === -1
+            && updateParameters.loggedIn === false
```

Comment:
because of the nature of this first trigger

https://github.com/ORCID/orcid-angular/blob/0e0f6d37a3ca435e868c828cb3a69b7dfdbfb5bf/src/app/core/user/user.service.ts#L140

Will never be true. 

Can you please double-check, isn't this bringing back the original error?

Reply:
Yes, it is bring it back other error. I will fix it